### ISSUE TITLE
Picker - PeoplePicker: Right to Left Support

### ISSUE THREAD

**Issue by Boris-Em** [OfficeDev/office-ui-fabric-react#2850]
### Bug Report

- __Package version(s)__:  5.0
- __Browser and OS versions__: FF, Chrome, Safari, Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Low

#### Describe the issue:

The Picker and PeoplePicker components open their drop down on the left side instead of the right side in RTL.

#### Actual behavior:

![screen shot 2017-09-14 at 2 07 06 pm](https://user-images.githubusercontent.com/4613643/30455844-4ed473ae-9956-11e7-9aaa-195c69cd0986.png)

#### Expected behavior:

The drop down menu of the Pickers should be opened on the right side.
